addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4597,38 +4578,24 @@ registry.sizing = SnippetOptionWidget.extend({
     async updateUIVisibility() {
         await this._super(...arguments);
 
-        const mobileViewThreshold = MEDIAS_BREAKPOINTS[SIZES.LG].minWidth;
-        const isMobileView = this.$target[0].ownerDocument.defaultView.frameElement.clientWidth < mobileViewThreshold;
-        const isGrid = this.$target[0].classList.contains('o_grid_item');
+        const isMobileView = weUtils.isMobileView(this.$target[0]);
+        const isGrid = !isMobileView && this.$target[0].classList.contains("o_grid_item");
         if (this.$target[0].parentNode && this.$target[0].parentNode.classList.contains('row')) {
             // Hiding/showing the correct resize handles if we are in grid mode
             // or not.
             for (const handleEl of this.$handles) {
                 const isGridHandle = handleEl.classList.contains('o_grid_handle');
                 handleEl.classList.toggle('d-none', isGrid ^ isGridHandle);
-                // Disabling the resize if we are in mobile view.
-                const isHorizontalSizing = handleEl.matches('.e, .w');
-                handleEl.classList.toggle('readonly', isMobileView && (isHorizontalSizing || isGridHandle));
```

Comment:
Sorry to interfere but I was curious (since you talked to me about it) so I took a look :slightly_smiling_face: 

If you remove these lines (at least concerning the grid handles) and these lines above 
```js
            // If the handle has the class 'readonly', don't allow to resize.
            // (For the grid handles when we are in mobile view).
            if (ev.currentTarget.classList.contains('readonly')) {
                return;
            }
```
the background grid is "displayed" in grid mode mobile view (see image) :grimacing: 

It is important to not remove the "return if readonly" because when resizing, ALL the sizing options are called one after the other (so sizing_x, sizing_y and sizing_grid) and this is the return that prevents a specific one to be called (so here, the return prevented to call the `onResize` of grid) :wink: Because you checked `isMobile` before displaying it, it is not visible but the space you see on the image is the background grid and it is updated in the `onResize` of sizing_grid 
I do not confirm what I say at 100% because I did not look deeply but this might explain why the resize in mobile view is a bit unstable (because both `onResize` are called successively) 

![image](https://github.com/odoo/odoo/assets/102796037/2b38a4ec-54af-4480-a2ea-b624d2f4d1c8)

Reply:
My bad, wrong guess, these line are not the cause... forget what I just said :sweat_smile: 
But in any case, the background grid should not be visible :wink:

Reply:
Also (last thing and then I will mind my own business :sweat_smile: ) the vertical sizing doesn't work in grid mode + mobile view (I think because this sizing manages the padding and the padding is set differently in grid mode so it has no impact -> to check)
=> maybe you should hide these handles in this case then ? :thinking: Or if not, then find a way to make both padding techniques coexist ?

Reply:
Padding resizing not working on mobile grid is the spec, but weird that the handles appear. Will check it too! Thanks.